### PR TITLE
Scheduling august

### PR DESCRIPTION
**Pull request by catsmith**
These are all the times from the August submissions.

### CODE REVIEW

**Comment by Chilipp** [comment]
Thanks @catsmith! @eileen-kuehn, could you please generate the registration forms?

**Comment by eileen-kuehn** [comment]
@Chilipp, thanks for letting me know! I created the events and also added a link back to the contribution on the website.

File: _events/talks/event-021.md [hunk 1]
```diff
@@ -4,11 +4,16 @@ authors:
...
       is_speaker: true
+time: 
+    - - start: 2020-11-10T10:00:00Z
+        end: 2020-11-10T10:30:00Z
 author: *speaker
 category: talks
 language: English
 prerequisites: "General software engineering principles"
 date: 2020-09-09
+last_modified_at: 2020-09-01
```
Comment (eileen-kuehn) [inline _events/talks/event-021.md:15]:
```suggestion
last_modified_at: 2020-09-15
```

File: _events/talks/event-026.md [hunk 2]
```diff
@@ -21,11 +21,16 @@ affiliations:
       index: 1
     - name: University of Notre Dame, US
       index: 2
+ time:
```
Comment (Chilipp) [inline _events/talks/event-026.md:24]:
```suggestion
time:
```

**Review by Chilipp — APPROVED** [review]

**Pull request merged**